CHANGELOG-Colang.md

```diff
@@ -30,6 +30,7 @@ Other:
 * [#594](https://github.com/NVIDIA/NeMo-Guardrails/pull/594) Add more information to Colang syntax errors.
 * [#599](https://github.com/NVIDIA/NeMo-Guardrails/pull/599) Runtime processing loop also consumes generated events before completion.
 * [#540](https://github.com/NVIDIA/NeMo-Guardrails/pull/540) LLM prompting improvements targeting `gpt-4o`.
+* [#669](https://github.com/NVIDIA/NeMo-Guardrails/pull/669) Merged (and removed) utils library file with core library.
```

Comment:
The `2.0-beta.2` version is now released, so this should go into the changelog for `2.0-beta.3`.

Reply:
In release 0.9.1.1? In this case we would need to adapt the Colang Changelog that still says `unreleased`.